### DISCUSSION TITLE
Position absolute and relative not being applied

### DISCUSSION THREAD

**Discussion by ssebastianoo** [tailwindlabs/tailwindcss#13462] [Help]
hi, I have this code:
```
 <div className='relative'>
      <div className='absolute inline-flex flex-col gap-2'>
```
but for some reason `position: absolute` and `position: relative` are not being applied to the elements

![image](https://github.com/tailwindlabs/tailwindcss/assets/48064277/232a4980-1ba9-4c3a-b581-0d64a6620de4)

**Comment by ssebastianoo** [comment]
Now I discovered that it's only for that page and that also padding and margin doesn't work. Height works only in some cases:

ex: `h-2` works but `h-[10px]` doesn't

**Comment by alex-krasikau** [comment]
Hey @ssebastianoo!

Seems to work fine for me - https://play.tailwindcss.com/r6LzRv8rcY. Could you please create a reproduction example in Play Tailwind, so we can help you resolve the issue?

**Reply by ssebastianoo** [reply]
Thanks for replying,
I'm going crazy, it works everywhere but on this file: https://github.com/ssebastianoo/pacmen/blob/main/src/pages/game/%5Bid%5D.jsx#L133

Maybe because that's the only .jsx file in a typescript project? I have no idea what's happening, the same code works on every other file (even on play.tailwindcss.com)

**Comment by ssebastianoo** [comment]
update: I got tired and rewrote everything in typescript (so it's now a `.tsx` and not `.jsx` anymore) and it's now working for some reason
